contracts/Lens/VenusLens.sol

```diff
@@ -58,6 +60,9 @@ contract VenusLens is ExponentialNoError {
             underlyingDecimals = EIP20Interface(vBep20.underlying()).decimals();
         }
 
+        ComptrollerInterface comptrollerInstance = ComptrollerInterface(comptrollerAddress);
+        uint venusSpeedPerBlock = comptrollerInstance.venusSpeeds(address(vToken));
```

Comment:
```suggestion
        uint venusSpeedPerBlock = comptroller.venusSpeeds(address(vToken));
```

Reply:
@kkirka 
do you mean to add comptrollerAddress to constructor args &  initialise comptrollerInstance inside constructor?
and
use it in all functions?

Reply:
You have `ComptrollerInterface comptroller = ComptrollerInterface(comptrollerAddress);` above, there's no need to define a second variable with the same value